.github/workflows/main.yaml

```diff
@@ -123,7 +123,7 @@ jobs:
       - name: Install dependences
         run: |
           command -v ginkgo || go get github.com/onsi/ginkgo/ginkgo
-          command -v kind || go get sigs.k8s.io/kind@v0.9.0
+          go get sigs.k8s.io/kind@v0.9.0
```

Comment:
I think root cause is https://github.com/kubeedge/kubeedge/blob/master/hack/lib/install.sh#L32, since in Github Action, we always install v0.9.

Reply:
So do we need to install kind in Github Action?

Reply:
Still need to avoid install failure since network issue.